app/controllers/api/v1/config_templates_controller.rb

```diff
@@ -95,7 +95,7 @@ def resource_class
       end
 
       def deprecated
-        ::ActiveSupport::Deprecation.warn('Config templates were renamed to provisioning templates')
+        Foreman::Deprecation.deprecation_warning("1.11", "Config templates were renamed to provisioning templates")
```

Comment:
We can exclude deleting deprecations from the v1 files. 
What you suggest on doing?